CONTRIBUTORS.md

```diff
@@ -0,0 +1,20 @@
+# Contributors
+
+The ODL maintainers are grateful to acknowledge contributions from several people. What follows is an incomplete list, see [contributors](https://github.com/odlgroup/odl/graphs/contributors) for a complete list and feel free to add yourself!
```

Comment:
More focused version:

We are grateful for contributions and would like to acknowledge all people who have helped improving the code and the documentation. What follows ...

Maybe the last part "feel free to add yourself" goes a bit too far. I see what you mean and I don't know a better formulation but this reads like "no requirements (not even being a contributor)".